website/src/blogs/how-to-install-openebs-on-ibm-cloud-private.md

```diff
@@ -0,0 +1,134 @@
+
+1. Log in to the ICP console and go to the **Admin/Repositories** menu.
+
+![OpenEBS on ICP](https://cdn-images-1.medium.com/max/800/0*PPZPNSr9_mW_9AZq.png)
+1. Click **Add repository**.
+
+![Add repository](https://cdn-images-1.medium.com/max/800/0*ZNaLIkk1gxFLWUJK.png)
+1. Add a chart repository with the following parameters:
+ — **Name:** openebs-charts
+ — **URL:**[https://openebs.github.io/charts/](https://openebs.github.io/charts/)
+
+![Confirm charts](https://cdn-images-1.medium.com/max/800/0*2m2J6V9YhnYk5_Cx.png)
+1. After you click **Add**, confirm that **openebs-charts** is listed under Repositories.
+
+![Catalog menu](https://cdn-images-1.medium.com/max/800/0*wkPxIB_Q2DevkgWh.png)
+1. Go to the **Catalog** menu, select **openebs** from the list.
+
+![Configure](https://cdn-images-1.medium.com/max/800/0*7Lt6IE4f_da0jZEB.png)
+1. On OpenEBS chart instructions page, click **Configure**.
+2. Configure OpenEBS deployment with the following parameters:
+ — **Release name:** openebs-<your-release-name> (you need to pick a unique name)
+ — **Target Namespace:** default (namespace should be the same as your workload)
+ — **rbacEnable:** true
+ — **image pullPolicy:** IfNotPresent
+ — **apiserver image:** openebs/m-apiserver
+ — **apiserver tag:** 0.4.0
+ — **provisione image:** openebs/openebs-k8s-provisioner
+ — **provisioner tag:** 0.4.0
+ — **jiva image:** openebs/jiva:0.4.0
+ — **replicas:** 2 (Number of Jiva Volume replicas)
+
+![Installation](https://cdn-images-1.medium.com/max/800/0*qfLs4pg_3TE1PbCB.png)
+1. Click **Install**. When finished click **View Helm Release.**
+
+![Storage classes](https://cdn-images-1.medium.com/max/800/0*raLyHiJeZ0hC_BAk.png)
+1. On the Helm Release page, you can see the status of OpenEBS, deployment, and available **Storage Classes**.
+
+![Deploy stateful application](https://cdn-images-1.medium.com/max/800/0*-gCAd374s2jXY3AP.jpg)
+1. Now, let’s try to deploy a stateful app on OpenEBS.
+
+### Install MongoDB on OpenEBS
+
+1. Under **Catalog**, select **ibm-mongodb-dev** and click **Configure**.
+2. Configure MongoDB deployment with the following parameters:
+ — **Release name:** mongodb-<your-release-name> (you need to pick a unique name here)
+ — **Target Namespace:** default (same as OpenEBS)
+ — **persistence enabled:** true
+ — **persistence useDynamicProvisioning:** true
+ — **dataVolume storageClassName:** openebs-mongodb
+ — **dataVolume size:** 2G (default is 20Gi, remove “i” — in current version it is not supported)
+ — **database password:** mongo
+ Accept the license agreements, keep all the other values as default and click **Install**.
+
+![Workloads release](https://cdn-images-1.medium.com/max/800/0*UTiLWk3zOy5bw_Wh.png)
+1. Go to **Workloads/Helm Releases** and select your MongoDB release. Under the **PersistentVolumeClaim** table you are going to see the volume claim and OpenEBS storage class.
+
+![Workloads deployment](https://cdn-images-1.medium.com/max/800/0*PNNp0nDxsZXzYwIH.png)
```

Comment:
After the image, we can have one line space for better formatting. Same comment for everywhere :)